apps/designer/app/designer/features/topbar/publish.tsx

```diff
@@ -15,15 +15,20 @@ import {
   useId,
 } from "@webstudio-is/design-system";
 import { useIsPublishDialogOpen } from "../../shared/nano-states";
+import env from "~/shared/env";
 
 type PublishButtonProps = { project: Project };
 
-const host =
-  typeof location === "object"
-    ? location.host.includes("webstudio.is")
-      ? "wstd.io"
-      : location.host
-    : "";
+const host = () => {
+  if (env.PUBLISHER_ENDPOINT && env.PUBLISHER_DOMAIN !== "") {
```

Comment:
both should be not empty strings and non null/undefined? or is it validated against null/undefined somewhere before?

Reply:
They are validated in the latest commit.